adapter/validator/src/main/java/org/mobilitydata/gtfsvalidator/validator/GtfsTypeValidator.java

```diff
@@ -95,39 +99,117 @@ public GtfsTypeValidator(@NotNull GtfsSpecificationProto.CsvSpecProto fileSchema
                     case INPUT_TYPE_UNSPECIFIED: // Text is default and does not require validation
                     case TEXT:
                     case COLOR:
-                    case DATE: //no special validation
+                    case DATE: // no special validation
                         break;
+                    case INTEGER:{
+                        if (!integerValidator.isInRange(
```

Comment:
that can never be true as in Integer by definition is within [`Integer.MIN_VALUE`, `Integer.MAX_VALUE`]